_playwright-tests/UI/RedHatRepo.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { describe } from 'node:test';
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Red Hat Repositories', () => {
```

Comment:
it would be better to change this to the describe method from playwright test (`test.describe()`) like in the other suites, that one works better with playwright adapters 😅

Reply:
oof yes missed changing this 😆 thank you!